github_connector/models/github_organization.py

```diff
@@ -0,0 +1,162 @@
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class GithubOrganization(models.Model):
+    _name = 'github.organization'
+    _inherit = ['abstract.github.model']
+    _order = 'name'
+
+    _github_type = 'organization'
+    _github_login_field = 'login'
+
+    # Columns Section
+    name = fields.Char(
+        string='Organization Name', required=True, readonly=True)
+
+    image = fields.Binary(string='Image', readonly=True)
+
+    description = fields.Char(string='Description', readonly=True)
+
+    email = fields.Char(string='Email', readonly=True)
+
+    website_url = fields.Char(string='Website URL', readonly=True)
+
+    location = fields.Char(string='Location', readonly=True)
+
+    ignored_repository_names = fields.Text(
+        string='Ignored Repositories', help="Set here repository names"
+        " you want to ignore. One repository per line."
+        " If set, the repositories will be created, but branches"
+        " synchronization and source code download will be disabled."
+        " Exemple:\n"
+        "odoo-community.org\nOCB\nOpenUpgrade\n")
```

Comment:
this is not the URL of a repository

Reply:
@elicoidal 
this is not a URL of a repository: they are three repository names, one repository by line, which they will be disabled to synchronize and download.

Reply:
"https://odoo-community.org/OCB/OpenUpgrade" is not a valid repository URL
You probably mean "https://github.com/OCB/OpenUpgrade"

Reply:
@elicoidal 
This is not an URL: they are repo names, one by line, for example:

purchase-workflow
sale-workflow
OCB

If you want I change odoo-community.org that is confusing for you.

Reply:
indeed I did not understand it at first. Still indeed the odoo-community.org is not a repository linked URL so I would suggest to change to something more logical